docs/application/dotnet/guides/text-input/voice-control-manager.md

```diff
@@ -0,0 +1,1073 @@
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+    To retrieve the commands you have added to the command list:
+
+    -   You can use the `GetCurrentCommands()` method of the [Tizen.Uix.VoiceControlManager.VoiceControlManagerClient](https://samsung.github.io/TizenFX/latest/api/Tizen.Uix.VoiceControlManager.VoiceControlManagerClient.html) class to get all commands from the command list:
+
+        ```
+        void ForeachCommand()
+        {
+            try
+            {
+                IEnumerable<VoiceCommand> command = VoiceControlManagerClient.GetCurrentCommands();
+            }
+            catch (Exception e)
+            {
+                /// Error handling
+            }
+        }
+        ```
+
+    -   You can use the `Commands` property of the `Tizen.Uix.VoiceControlManager.VoiceCommandsGroup` class to get the current commands in an output parameter.
+        ```
+        void GetCommands()
+        {
+            try
+            {
+                VoiceCommand current_command = new VoiceCommand();
+                VoiceCommandsGroup group = new VoiceCommandsGroup();
+
+                /// Other actions
+
+                current_commands = group.Commands;
+            }
+            catch (Exception e)
+            {
+                /// Error handling
+            }
+        }
+        ```
+
+4.  Register the commands for recognition by setting the command group to the voice control.
+
+    If you want to update registered commands, set the command group again with the updated commands.
+
+    ```
+    void SetCommand()
+    {
+        try
+        {
+            VoiceCommandsGroup group = new VoiceCommandsGroup();
+            VoiceCommand command = new VoiceCommand();
+            command.Command = "test";
+            command.Type = CommandType.Foreground;
+
+            group.Commands.Add(command);
+
+            VoiceControlManagerClient.SetCommands(group);
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+5.  When no longer needed, unset the command group:
+
+    ```
+    /// Unset the command group
+    void UnsetCommand()
+    {
+        try
+        {
+            VoiceCommandsGroup group = new VoiceCommandsGroup();
+            VoiceCommand command = new VoiceCommand();
+            command.Command = "test";
+            command.Type = CommandType.Foreground;
+
+            group.Commands.Remove(command);
+
+            VoiceControlManagerClient.SetCommands(group);
+            ///or
+            VoiceControlManagerClient.ClearCommands();
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+
+
+<a name="other_commands"></a>
+## Other command functions
+
+- Register the commands for recognition by setting the command group to the voice control from file.
+    If you want to update registered commands, set the command group again with the updated commands.
```

Comment:
Please use colon at the end of the sentence as it is following with code block. Also suggesting rephrase as:

If you want to update registered commands....